interfaces/seccomp/backend.go

```diff
@@ -313,10 +314,10 @@ func generateContent(opts interfaces.ConfinementOptions, snippetForTag string, a
 	buffer.WriteString(snippetForTag)
 	buffer.WriteString(uidGidChownSyscalls)
 
-	// For systems with force-devmode we need to apply a workaround
-	// to avoid failing hooks. See description in template.go for
-	// more details.
-	if release.ReleaseInfo.ForceDevMode() {
+	// For systems with partial or missing AppArmor support we need to apply
```

Comment:
This is interesting in light of cgroup v2 work, where we are no longer in full confinement mode when v2 is used. I think this is a step in the correct direction.